client/components/services/useContactInfo.tsx

```diff
@@ -0,0 +1,68 @@
+import { getUserId, useAuthContext } from '@contexts';
+import axios from 'axios';
+import { useFormContext } from './useFormContext';
+import { useState, useEffect } from 'react';
+
+const initialValuesInit = {
+  facilityName: '',
+  applicantName: '',
+  primaryContactName: '',
+  phoneNumber: '',
+  mailingAddress: '',
+  mailingAddressPostalCode: '',
+  isOneApplication: '',
+  priority: '',
+};
+
+export const useContactInfo = () => {
+  // maybe move this to context later
+  const userId = getUserId();
+  const keycloak = useAuthContext().keycloak;
+
+  const [initialValues, setInitialValues] = useState(initialValuesInit);
+  const { updateProceedToNext } = useFormContext();
+  const [applicationId, setApplicationId] = useState('');
+
+  useEffect(() => {
+    const getData = async () => {
+      const data = { userId: userId };
+
+      const options = {
+        method: 'GET',
+        headers: {
+          'content-type': 'application/x-www-form-urlencoded',
+          Authorization: `Bearer ${keycloak?.idToken}`,
+        },
+        data,
+        url: 'http://localhost:8080/api/v1/applications/in-progress',
```

Comment:
Shouldn't hardcode the API_URL. - Move all the generic components like headers, url outside the scope of this fn.

/applications/in-progress - to be pushed to a constants file

Reply:
Also, the API call should be to `/applications/applicationId` -> the applicationId of the form should be fetched in the context and set before entering this screen.